src/Files.App/Views/Properties/MainPropertiesPage.xaml

```diff
@@ -103,8 +144,10 @@
 						<RowDefinition Height="52" />
 					</Grid.RowDefinitions>
 
-					<Frame x:Name="contentFrame" IsNavigationStackEnabled="False" />
+					<!--  Frame  -->
+					<Frame x:Name="MainContentFrame" IsNavigationStackEnabled="False" />
 
+					<!--  Save/Cancel buttons  -->
 					<Grid
 						Grid.Row="1"
 						Background="{ThemeResource CardBackgroundFillColorSecondary}"
```

Comment:
Do you want to remove the background to match the design concept?
![image](https://user-images.githubusercontent.com/39923744/230944280-f230cb28-4a16-45e1-b012-f26ded0ebd14.png)

Reply:
Save/Cancel buttons should be on the same layer as NavigationView?

Reply:
It's the suggested design but we don't have to follow it. I'll leave it to you to decide.

Reply:
I won't